consensus/polybft/consensus_runtime.go

```diff
@@ -39,6 +39,18 @@ type Transport interface {
 	Gossip(message interface{})
 }
 
+// txPoolInterface is an abstraction of transaction pool
+type TxPoolInterface interface {
```

Comment:
Seems like there is no need to export this interface (it is solely used by `consensus/polybft` package)